app/population/controllers/edit.js

```diff
@@ -50,6 +52,19 @@ class DetailsController extends FormWizardController {
     }
   }
 
+  setBreadcrumbs(req, res, next) {
+    const breadcrumbText = req.population
```

Comment:
This looks identical to the condition in `setPageTitle`. Could this be extracted?

Reply:
It's only the 2nd instance of duplication - I find prematurely abstracting to be an anti-pattern and leads to more confusing code. Once the code is repeated 3 or 4 times, I think there's an argument for reuse.